examples/grpc-bridge/config/s2s-python-envoy.json

```diff
@@ -0,0 +1,74 @@
+{
+  "listeners": [
+    {
+      "port": 9001,
+      "filters": [
+        {
+          "type": "read",
+          "name": "http_connection_manager",
+          "config": {
+            "codec_type": "auto",
+            "add_user_agent": true,
+            "idle_timeout_s": 840,
+            "access_log": [
+              {
+                "path": "/var/log/envoy/egress_http_error.log",
```

Comment:
@RomanDzhabarov I think it would be interesting for people to see requests flowing out of the python container and into the grpc one. But, your comment made me think that if that is the purpose of keeping the log here, we should add a log in the ingress in the gRPC config.